docs/user_guide/building_blocks/config_objects.md

```diff
@@ -53,9 +121,340 @@ given via the ```--variables``` argument, accessing information from columns bas
 the `expression` of the Variable and storing them in histograms with binning defined
 via the `binning` argument of the Variable.
 
-The list of possible keyword arguments can be found in
-Variable. The values in the ```expression``` argument can
+The list of possible keyword arguments can be found in the order documentation for the class
+{external+order:py:class}`order.variable.Variable`. The values in the ```expression``` argument can
 be either a one-dimensional or a more dimensional array. In this second case the information is
 flattened before plotting. It is to be mentioned that
 {py:attr}`~columnflow.columnar_util.EMPTY_FLOAT` is a columnflow internal null value and
 corresponds to the value ```-9999.0```.
+
+### Category
```

Comment:
This section should also link to the Categories user guide (which is empty for now, but I already started working on that)

Reply:
done